crates/strategies/blend-strategies/src/auctioneer_strategy.rs

```diff
@@ -698,26 +693,23 @@ impl BlendAuctioneer {
     /// Process new block events, updating the internal state.
     async fn process_new_block_event(&mut self, event: NewBlock) -> Result<Vec<Action>> {
         let mut actions = Vec::new();
-        let mut assets = self.supported_collateral.clone();
-        assets.extend(self.supported_liabilities.clone());
         if event.number % 10 == 0 {
             get_asset_prices_db(
                 &self.rpc,
                 &self.oracle_id,
                 &self.oracle_decimals,
-                &assets,
+                &self.supported_liabilities,
```

Comment:
I believe this needs to include base collateral right?

Reply:
Base collateral is required to exist in supported liabilities at this point. We should include a documentation update to make this obvious.

- [ ] Include documentation